web/html/src/manager/virtualization/nets/properties/VirtualPortFields.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import * as React from 'react';
+import Validation from 'components/validation';
+import { Radio } from 'components/input/Radio';
+import { Text } from 'components/input/Text';
+import { FormContext } from 'components/input/Form';
+import * as utils from './utils';
+
+type Props = {
+  type: string | void | null,
+}
+
+export function VirtualPortFields(props: Props) {
+  const formContext = React.useContext(FormContext);
+  const model = formContext.model || {};
+  return (
+    <>
+      { props.type === "802.1qbh" && (
+        <Radio
+          name="virtualport-params-type"
+          inline={true}
+          label={t('Parameters')}
+          required
+          labelClass="col-md-3"
+          divClass="col-md-6"
+          defaultValue='profileid'
+          items={[
+            {label: t('By profile id'), value: 'profileid'},
+            {label: t('Virtual Station Interface (VSI) parameters'), value: 'vsi'},
+          ]}
+        />)
+      }
+      { ("openvswitch" === props.type || ("802.1qbh" === props.type && model['virtualport-params-type'] === "profileid")) && (
```

Comment:
Very small nitpick, I don't really have a strong preference on which way it is, please prefer either `props.type === "foo"` or `"foo" === props.type` consistently, right now it's one way in one place and the other way in the other.

Reply:
ACK